modules/objdetect/test/test_boarddetection.cpp

```diff
@@ -318,4 +318,12 @@ TEST(CV_ArucoGenerateBoard, regression_1226) {
     });
 }
 
+TEST(CV_ArucoDictionary, extendDictionary) {
+    aruco::Dictionary base_dictionary = aruco::getPredefinedDictionary(aruco::DICT_4X4_250);
+    ASSERT_EQ(base_dictionary.bytesList.rows, 250);
```

Comment:
The assertion does not make sense. It's constant in OpenCV with known size.

Reply:
removed